src/css/app.scss

```diff
@@ -217,3 +217,84 @@ div.cleos-auth-dialog {
   text-overflow: ellipsis;
 }
 
+// TODO: refactoring needed
+// https://github.com/telosnetwork/telos-wallet/issues/254
+
+.c-notify {
+    width: 100%;
+    max-width: 400px;
+    margin: 1rem;
+    padding: 0;
+    border-radius: 4px;
+    box-shadow: 0 0 20px 0 rgba(0, 0, 0, 0.2);
+    background-color: var(--bg-color, #fff);
+    transition: all 0.3s ease;
+    overflow: hidden;
+    // we remove parent padding
+    .q-notification__message {
+        padding: 0;
+    }
+    // we need to style the action buttons
+    .q-notification__actions {
+        justify-content: space-around;
+        padding: 24px;
+        .q-btn {
+            font-weight: bold;
+            &.text-positive {
+                i {
+                    margin-left: 0px;
+                    margin-top: -5px;
+                    transform: scale(0.5);
+                }
+            }
+        }
+    }
+    // container will say if this notification is success or error and set the color
+    &__container {
+        &--success {
+            --notify-color: #{$notify-success};
+        }
+        &--error {
+            --notify-color: #{$notify-error};
+        }
+        display: flex;
+        flex-direction: column;
+    }
+    &__header {
+        height: 8px;
+        background-color: var(--notify-color);
+    }
+    &__title {
+        display: flex;
+        align-items: center;
+        padding: 1rem;
+        justify-content: center;
+        text-transform: uppercase;
+        font-size: 24px;
+        svg {
+            width: 2.1rem;
+            height: 2.1rem;
+            margin-right: 0.3rem;
+        }
+        span {
+            color:  var(--notify-color);
+            font-weight: 600;
+        }
+    }
+    &__message {
+        color: var(--text-color, #000);
+        padding: 0 2rem;
+        text-align: center;
+        span {
+            font-size: 16px;
+        }
+    }
+    &__action-btn {
+        color: var(--text-color, #000);
+        &--hide {
+            color: transparent;
+            background-color: transparent;
+            pointer-events: none;
+        }
+    }
+}
```

Comment:
@EJSG and @donnyquixotic
This is the second part of the c-notify "component" (which is not a component)

I need some opinions on where to put this code.

Reply:
its totally normal in my opinion to need to have styles outside of a component sometimes. the way this should be handled, i think, is to have a new folder in the css directory called `components` with a new file for each logical component. so in this case it would be a new file `css/components/_notification.scss`. then i think you can add a line to the main app sass file like `@import 'components/notification'